components/admin_console/admin_user_card/admin_user_card.jsx

```diff
@@ -31,6 +31,9 @@ const AdminUserCard = (props) => (
                 <Bullet user={props.user}/>
                 <span className='AdminUserCard__user-nickname'>{props.user.nickname}</span>
             </div>
+            <div className='AdminUserCard__user-id'>
+                {'User ID:'} {props.user.id}
```

Comment:
```suggestion
                {Utils.localizeMessage('admin.userManagement.userDetail.userId', 'User ID:')} {props.user.id}
```

Reply:
You can import the `localizeMessage` function via `import * as Utils from 'utils/utils.jsx';`

Reply:
Be sure to run `make extract-i18n` which should populate the `en.json` translation file with the default string `User ID:`.

Reply:
I did that when I noticed CI failed, but ran into some issue. Hopefully it will work this time!